app/services/decidim/verifications/sant_boi_census/sant_boi_census_authorization_handler.rb

```diff
@@ -82,8 +82,6 @@ def sanitized_document_type
         #   status => WS response status, as Integer
         def perform_request
           SantBoiCensusAuthorizationService.new(sanitized_document_type, document_number).perform_request
-        rescue StandardError
```

Comment:
if you remove the rescue, what will happen when the service raises an exception?

Reply:
The exception would be rescued in `SantBoiCensusAuthorizationService#perform_request`

```ruby
# app/services/decidim/verifications/sant_boi_census/sant_boi_census_authorization_service.rb:31
rescue StandardError => e
  Rails.logger.error "[#{self.class.name}] Failed to perform request"
  Rails.logger.error e.message
  Rails.logger.error e.backtrace.join("\n")
  {} # To avoid crashing in SantBoiCensusAuthorizationHandler::ws_request_must_succeed
end
```

And a single error would be added in `SantBoiCensusAuthorizationHandler#ws_request_must_succeed`

```ruby
# app/services/decidim/verifications/sant_boi_census/sant_boi_census_authorization_handler.rb:40
def ws_request_must_succeed
  return if errors.any?

  @response = perform_request

  errors.add(:base, I18n.t("connection_failed", scope: "errors.messages.sant_boi_census_authorization_handler")) unless @response[:status] == 200
end
```

Reply:
ah ok, I thought sant_boi_census_authorization_service could raise an exception